docs/book/introduction.md

```diff
@@ -16,15 +16,135 @@ description: Welcome to ZenML!
 
 If you answered yes to any of the questions above, ZenML is here to help you:
 
-* **ZenML** is an **open-source MLOps framework** built for data scientists, ML Engineers, and MLOps Developers.
-* It comes equipped with a simple and flexible **Python SDK** that helps you to create **portable and production-ready pipelines**.
-* It also features **cloud-** and **tool-agnostic** **interfaces/abstractions** such as orchestration systems, artifact repositories, and model deployment platforms that cover the entire ML journey.
-* With these abstractions, it remains **extensible** and **open to customization**. That is also backed by numerous **collaborative features** that allow you to work not just with your team but also with the community.
-* Moreover, this creates a separation between the different aspects of ML workflows, allowing users to focus on creating their own respective solutions in an efficient manner.
-* Overall, it reduces the complexity of your development lifecycle, all the way **from an experimentation environment to a production-grade solution**.
+**ZenML** is an extensible, open-source MLOps framework for creating portable, 
+production-ready MLOps pipelines. 
+
+By decoupling infrastructure from code, ZenML gives all ML developers in your 
+organization more freedom and independence in how they approach their work:
+
+![ZenML Overview](.gitbook/assets/intro_zenml_overview.png)
+
+{% tabs %}
+
+{% tab title="ZenML for Data Scientists" %}
+
+As a Data Scientists, ZenML gives you the freedom to focus on modeling and
+experimentation without having to worry about production issues at all.
+
+**Tooling Freedom:** Since ZenML code runs in any environment, you can 
+develop ML models locally with all your favourite tools. Once you are happy with 
+your results, you can switch to a production environment via a single command 
+and all your code will just work:
+
+```bash
+python run.py  # develop your code locally with all your favourite tools
+zenml stack set production
+python run.py  # run your code in production without any code changes
+```
+
+**Pythonic SDK:** ZenML is designed to be as unintrusive as possible. Adding 
```

Comment:
Based on the look of the page, how about putting the 3 points in each tab into lists? Something like:

> As a ....., .........
> 
> - Tooling Freedom: ....
> - Reuseability: ...

Reply:
Yeah I had that before but it looks weird because the items will be indented but the code snippets are not

Reply:
As far as I know, if you like the lists, we can indent the code blocks afterward in GitBook.